packages/tools/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this tool will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Changed
+- Increase SVG file size limit in rollup configuration from 16kb to 100kb
```

Comment:
```suggestion
- Increase the SVG file size limit from 14 kB to 100 kB for the structure preview of widgets. 
```

Reply:
The changelog shouldn't be to technical. You should see it as the release notes for PIW tools.